main.go

```diff
@@ -2,15 +2,45 @@ package main
 
 import (
 	"log"
+	"net/http"
 	"os"
 
 	"github.com/apex/gateway"
+	"github.com/gin-contrib/static"
 	"github.com/gin-gonic/gin"
+	"github.com/gin-gonic/gin/binding"
 )
 
+type contactDetails struct {
+	Name 				string `json:"name"`
+	Designation 		string `json:"designation"`
+	designationInfo 	string `json:"designation_info"`
```

Comment:
Struct field names should start with a capital letters, And also run `go fmt .` for formatting go code